Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```diff
@@ -2896,6 +2896,672 @@ script:
+      description: The agent identifier of the agent that quarantined the file.
+      type: String
+    - contextPath: CrowdStrike.QuarantinedFile.cid
+      description: The unique identifier for the customer that owns the agent.
+      type: String
+    - contextPath: CrowdStrike.QuarantinedFile.sha256
+      description: The SHA256 hash value of the quarantined file.
+      type: String
+    - contextPath: CrowdStrike.QuarantinedFile.paths.path
+      description: The full path of the quarantined file.
+      type: String
+    - contextPath: CrowdStrike.QuarantinedFile.paths.filename
+      description: The name of the quarantined file.
+      type: String
+    - contextPath: CrowdStrike.QuarantinedFile.paths.state
+      description: The current state of the quarantined file path (e.g., "purged").
+      type: String
+    - contextPath: CrowdStrike.QuarantinedFile.state
+      description: The current state of the quarantined file (e.g., "unrelease_pending").
+      type: String
+    - contextPath: CrowdStrike.QuarantinedFile.detect_ids
+      description: The detection identifiers associated with the quarantined file.
+      type: String
+    - contextPath: CrowdStrike.QuarantinedFile.hostname
+      description: The hostname of the agent that quarantined the file.
+      type: String
+    - contextPath: CrowdStrike.QuarantinedFile.username
+      description: The username associated with the quarantined file.
+      type: String
+    - contextPath: CrowdStrike.QuarantinedFile.date_updated
+      description: The date the quarantined file was last updated.
+      type: Date
+    - contextPath: CrowdStrike.QuarantinedFile.date_created
+      description: The date the quarantined file was created.
+      type: Date
+  - arguments:
+    - description: The IDs of the quarantined files to update.
+      name: ids
+      required: false
+      isArray: true
+    - auto: PREDEFINED
+      description: Action to perform against the quarantined file.
+      name: action
+      required: true
+      predefined:
+      - delete
+      - release
+      - unrelease
+    - description: Comment to list along with action taken.
+      name: comment
+      required: true
+    - description: Update files based on a custom filter.
+      name: filter
+      required: false
+    - description: Update files based on sha256.
+      name: sha256
+      required: false
+      isArray: true
+    - description: Update files based on file name.
+      name: filename
+      required: false
+      isArray: true
+    - description: Update files based on state.
+      name: state
+      required: false
+    - description: Update files based on host name.
+      name: hostname
+      required: false
+      isArray: true
+    - description: Update files based on user names.
+      name: username
+      required: false
+      isArray: true
+    description: Update quarantine file metadata by specified ids or a filter.
+    name: cs-falcon-apply-quarantine-file-action
```

Comment:
Should this be quarantine or quarantined?

Reply:
@meital-aizen What do you think?

Reply:
@MosheEichler The description should be `Apply action to quarantined file by file ids or filter`.